drivers/serial/uart_scif.c

```diff
@@ -1246,14 +1244,14 @@ static const struct uart_driver_api uart_scif_driver_api = {
 		UART_SET_IRQ(n, txi);							\
 		UART_SET_IRQ(n, tei);							\
 	}
-#define UART_SCIF_IRQ_CFG_FUNC_INIT(n) 							\
+#define UART_SCIF_IRQ_CFG_FUNC_INIT(n)							\
 	.irq_config_func = irq_config_func_##n
-#define UART_SCIF_INIT_CFG(n, soc_type) 						\
+#define UART_SCIF_INIT_CFG(n, soc_type)						\
```

Comment:
Is this Github displaying trailing slash incorrectly, or you are having issues with formatting here?

Reply:
This was caused because Tabs were used as an offset before "\". Zephyr accepts both tabs and spaces. But I prefer spaces because when using tabs - format can look different depending of the editor mode. Clang format suggests spaces.
I've decided to leave tabs, but removed redundant spaces to avoid huge changeset.
Right now I've added extra tab to to PR so github should display correctly.